include/net/net_pkt.h

```diff
@@ -1184,6 +1184,21 @@ int net_pkt_get_dst_addr(struct net_pkt *pkt,
 void net_pkt_get_info(struct k_mem_slab **pkts,
 		      struct net_buf_pool **data);
 
+/**
+ * @brief Convert net_pkt pointer to data pointer, cast to the specified type.
+ *
+ * Note: Ensure first, that there is enough contiguous data in net_pkt
+ *       with net_pkt_pullup().
+ */
+#define net_pkt_tod(_pkt, _type) ((_type)((_pkt)->iter.pos))
+
+/**
+ * @brief Make first len bytes contiguous.
+ * @return On error returns NULL and net_pkt is unrefenced,
+ *         net_pkt pointer otherwise.
+ */
+struct net_pkt *net_pkt_pullup(struct net_pkt *pkt, int len);
```

Comment:
The documentation needs fixing meaning that parameters needs to be documented, sanitychecker will complain about these otherwise.

Reply:
Thanks, will fix.